posts/2016-09-20-IDA2016-notes.md

```diff
@@ -0,0 +1,126 @@
+
+## Research Data Alliance (RDA) 8th Plenary
+
+The RDA plenary had sessions for working groups, interest groups, and birds of a feather groups (informal, single-time events). Each of these sessions involved some presentation with plenty of time for discussion.
+
+Research Data Alliance is a collection of academics, librarians, and other groups involved with research data. RDA has a [good primer](https://rd-alliance.org/rda-newcomers-some-notes) on their organization structure. RDA members meet with a focus on interest groups and working groups. Working groups and interest groups continue to communicate throughout the year to form recommendations (working groups) or help inform discussion throughout RDA (interest groups).
+
+### Session Notes
+
+Specific sessions notes collected below.
+
+#### [Long Tail Data IG Meeting](https://rd-alliance.org/ig-long-tail-research-data-rda-8th-plenary-meeting)
+
+* The [interest group of long tail data](https://rd-alliance.org/groups/long-tail-research-data-ig.html) is very relevant to Dat. They are [drafting these recommendations](https://docs.google.com/document/d/1TL2VbuIs8mHDT4MZyK3jqkdSP7cL9O7GbMbapu_9CNQ/edit).
+* Amazon succeeded by providing access to the long tail of books. There is a huge gap is services/infrastructure for the long tail of data. This long tail is from individual researchers. Generally small data (>1GB). **90% of researchers keep data on local, lab, or institution servers/backups, not in data repositories.** 
+* Most data isn't highly structured or doesn't have standardized metadata. But this is where a lot of the innovation comes from. Newer research fields, newer types of data, etc. These data will be impossible to put in many data repositories.
+* "Important to show immediate value upon deposit to a data repository".
+* "Data needs to link to other resources".
+* "Data that doesn't lead to publication is the data that currently gets lost".
+* [Big data, little data, no data](https://mitpress.mit.edu/big-data-little-data-no-data) - Book that may be of interest.
+* Orphan data are the bigger problem, data that don't have a home.
+* Libraries will provide for a key partner in managing and publishing long tail data.
+* Metadata is important but need to strike balance between ease of use and standardization/discovery.
+
+#### [Joint meeting of IG Preservation e-Infrastructure, IG Reproducibility: Knowledge Preservation Tools for Researchers](https://rd-alliance.org/rda-8th-plenary-joint-meeting-ig-preservation-e-infrastructure%C2%A0ig-reproducibility)
+
+* How can we foster relationships in overlap between researcher and archivist? Bridge the gap between researcher with data and preservation needs. What tools are necessary to make this easier?
```

Comment:
this section starts turning into a bulleted list which is totally acceptable but a bit abrupt coming from the section before which is in paragraph form